Automattic/wp-calypso
# Plans: Show fewer features in the grid

Related to [#3332](https://github.com/Automattic/martech/issues/3332) 

## Proposed Changes

Updates the features grid on `/start/plans` and `/plans/:site` to fix inaccuracies, and to display fewer features per plan so that differentiating features are visible above the fold. Further context in the project thread (p5uIfZ-fIN-p2).

This experiment includes two treatment variants:
  
- Treatment A (focuses on fixing inaccuracies)
  - Some features added
  - Many features removed
  - Updated titles and descriptions for various features
  - Ordering of features has changed
  
- Treatment B (extends Treatment A, focuses on displaying less features)
  - More features removed
  - Vertical alignment of all features
  - Removed the copy: "Everything in xx, plus"
  - Larger font size for features
 
For details on how we are measuring the impact of these changes, please review the experiment post (pbxNRc-45y-p2).

---

**Treatment A**

| Control | Treatment |
| -- | -- |
|<img size=400 src="https://github.com/user-attachments/assets/e14098d0-33a3-487b-8b87-db2faa47974d" />|<img size=400 src="https://github.com/user-attachments/assets/0a8953a8-21ed-4f03-8d23-d6a15cecd156" />|

---

**Treatment B**

| Control | Treatment |
| -- | -- |
|<img size=400 src="https://github.com/user-attachments/assets/e2c6f299-40e5-4dcf-8820-20f9d0250725" />|<img size=400 src="https://github.com/user-attachments/assets/bb34c2d8-7950-411d-89f8-d626d6423be9" />|



## Why are these changes being made?


The hypothesis is that a simplified features grid, displaying differentiators above the fold (pbxNRc-3Tw-p2), will help users find the right plan and have a positive impact on sign-ups and cash sales. 

## Testing Instructions



Please verify the following:
  - The treatment variants appear as per the above screenshots
  - The changes are limited to `/start/plans` and `/plans/:site`
  - The control variant appears as per production (ie. no changes)

Treatment A can be tested with the flag `simplified-features-grid-a`:
* `/start/plans?flags=simplified-features-grid-a`
* `/plans/:site?flags=simplified-features-grid-a`

Treatment B can be tested with the flag `simplified-features-grid-b`:
* `/start/plans?flags=simplified-features-grid-b`
* `/plans/:site?flags=simplified-features-grid-b`

---

Stories have also been added for each treatment variant:

- 'Experiment [Simplified features grid]: Treatment A'
- 'Experiment [Simplified features grid]: Treatment B'

<img size=400 src="https://github.com/user-attachments/assets/d2a2f33b-39b5-4f9b-8376-b642fcd312cd" />





## Pre-merge Checklist
